src/Button/sgds-button.ts

```diff
@@ -38,6 +59,60 @@ export class SgdsButton extends LitElement {
   /** Tells the browser to download the linked file as this filename. Only used when `href` is set. */
   @property({ reflect: true }) download?: string;
 
+  /**
+   * The "form owner" to associate the button with. If omitted, the closest containing form will be used instead. The
+   * value of this attribute must be an id of a form in the same document or shadow root as the button.
+   */
+   @property() form: string;
+
+   /** Used to override the form owner's `action` attribute. */
+   @property({ attribute: 'formaction' }) formAction: string;
+ 
+   /** Used to override the form owner's `method` attribute.  */
+   @property({ attribute: 'formmethod' }) formMethod: 'post' | 'get';
+ 
+   /** Used to override the form owner's `novalidate` attribute. */
+   @property({ attribute: 'formnovalidate', type: Boolean }) formNoValidate: boolean;
+ 
+   /** Used to override the form owner's `target` attribute. */
+   @property({ attribute: 'formtarget' }) formTarget: '_self' | '_blank' | '_parent' | '_top' | string;
+
```

Comment:
Hihi just want to understand where this the form attributes used and how it is working, maybe we can clarify during the standup later ?

Reply:
these are the props u can pass into the button to overwrite certain attributes to the `<form>` wrapper.

Reply:
I've added the link to  mdn's  `<form>` page in the storybook for further details